### PR TITLE
feat(ui): add news carousel

### LINKED ISSUE #69: [UI] News carousel
The carousel on top.
 Design: [Figma](https://www.figma.com/file/GjOEDxaZ4sT9mBr8A10IiA/Gymkhana?node-id=2%3A2)

- Hide images on small screens, i.e. only display info card.
- All the data will come from a list. Each list element will have all the fields to display a carousel slide.

### PR DESCRIPTION
**Pull request by Aksh-Bansal-dev**
closes #69

### CODE REVIEW

**Comment by Aksh-Bansal-dev** [comment]
@Samy-33 can you review this?

File: ui/src/components/NewsCarousel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    clubName: 'The Programming Club',
+    description:
+      'Bayerische Motoren Werke AG, commonly referred to as BMW is a German multinational corporate manufactu',
+    imgUrl: dummyNewsImg,
+  },
+];
+
+export const NewsCarousel: React.FC = () => {
+  return (
+    <Carousel
```
Comment (Samy-33) [inline ui/src/components/NewsCarousel/index.tsx:32]: When the data is empty it should should show no events yet at the center.

**Review by Samy-33 — CHANGES_REQUESTED** [review]
Looks good overall, just a small change.

File: ui/src/components/NewsCarousel/index.tsx (hunk 1 above)
Comment (Aksh-Bansal-dev) [reply ui/src/components/NewsCarousel/index.tsx:32]: When the data is empty, this component does not render anything.

Comment (Samy-33) [reply ui/src/components/NewsCarousel/index.tsx:32]: I see. Yeah, I think we can do away with this element if there are no highlighted events.

**Review by Samy-33 — APPROVED** [review]
LGTM.

**Pull request merged**